jjb/anax/anax.yml

```diff
@@ -0,0 +1,21 @@
+- job:
+    name: jenkins-job
+    project-type: pipeline
+    defaults: global
+    node: 'ubuntu2204-builder-aws-1c-2g'
```

Comment:
This will be ignored. It needs to be defined in the pipeline as the "agent". The agent can be defined in the JJB code as a parameter, but that'll need to be passed to the Jenkinsfile.